api/v1alpha5/metal3machine_types.go

```diff
@@ -62,6 +62,12 @@ type Metal3MachineSpec struct {
 	// NetworkData is an object storing the reference to the secret containing the
 	// network data given by the user.
 	NetworkData *corev1.SecretReference `json:"networkData,omitempty"`
+
+	// When set to disabled, automated cleaning will be skipped
+	// during provisioning and deprovisioning.
+	// +kubebuilder:default:=metadata
+	// +kubebuilder:validation:Enum:=metadata;disabled
+	AutomatedCleaningMode string `json:"automatedCleaningMode,omitempty"`
```

Comment:
same as above, Field name is not descriptive.
`AutomatedDiskCleaningMode` ?

Reply:
This is has been already implemented as `AutomatedDiskCleaningMode`, so changing it here means I need to replace it in BMO as well and in release-0.4 branch. I agree that it might not fully explain what it does, but for that reason I've added documentation.